.gitignore

```diff
@@ -5,3 +5,6 @@ symbols_cache/
 
 .docker-build
 my.env
+.env
+uv.lock
+.venv
```

Comment:
I'm switching my whole world to use `uv` and `just`, so this is related to that. It shouldn't affect anyone else.